docs/flags.md

```diff
@@ -89,12 +118,19 @@ export class MyCLI extends Command {
 
   async run() {
     const {flags} = await this.parse(MyCLI)
-    if (flags.team) console.log(`--team is ${flags.team}`)
+    if (flags.team) console.log(`--team is ${flags.team.name}`)
   }
 }
 ```
 
-In the Heroku CLI, we use flags for our `--app` flag which takes advantage of a lot of functionality. It can be useful [to see how that is done](https://github.com/heroku/cli-engine-heroku/blob/oclif/src/flags/app.ts) to get ideas for making custom flags.
+In the Salesforce CLI we make heavy use of custom flags. For example,
+
+- A [`salesforceId`](https://salesforcecli.github.io/sf-plugins-core/globals.html#salesforceidflag) flag that ensures the provided string is a valid Salesforce Id.
+- A [`duration`](https://salesforcecli.github.io/sf-plugins-core/globals.html#durationflag) flag that converts a provided integer into a `Duration` instance that we use for working with time based values.
+
+These and more are located [here](https://github.com/salesforcecli/sf-plugins-core) if you want to see more examples. You can also read the [API docs](https://salesforcecli.github.io/sf-plugins-core/globals.html).
```

Comment:
Let link directly to the `flags` dir: https://github.com/salesforcecli/sf-plugins-core/tree/main/src/flags